### DISCUSSION TITLE
ravynos & hellosystem

### DISCUSSION THREAD

**Discussion by keyusb** [ravynsoft/ravynos#425] [General]
It seems to me that the philosophy of the system, although a bit different, can be compared to that of HelloSystem: why don't we join forces instead of making similar systems that disperse resources? Aren't you making the same mistake as linux with a multiplicity of distributions but that none of them are a real solid alternative?

**Comment by mszoek** [comment]
Hi! This is a FAQ which has an answer here: https://ravynos.com/faq

I was working with probono initially on some things. There is not as much alignment in goals as you probably think.
